src/org/jitsi/impl/neomedia/rtp/StreamRTPManager.java

```diff
@@ -168,14 +171,17 @@ public void putResumableStreamRewriter(
     public ResumableStreamRewriter getResumableStreamRewriter(Long ssrc,
                                                               boolean create)
     {
-        ResumableStreamRewriter rewriter = ssrcToRewriter.get(ssrc);
-        if (rewriter == null && create)
+        synchronized (ssrcToRewriter)
```

Comment:
That sounds sensible, but the method is used for every incoming RTP/RTCP packet, so I'd rather avoid introducing synchronization for obvious reasons. The JVB accesses this method sequentially, before the OutputDataStreamImpl.